pkgs/development/compilers/open-watcom/bin.nix

```diff
@@ -87,6 +87,7 @@ in
 stdenvNoCC.mkDerivation rec {
   pname = "open-watcom-bin";
   version = "1.9";
+  name = "${pname}-unwrapped-${version}";
```

Comment:
You can drop this, if you set `name` to `"open-watcom-bin-unwrapped"`. Same in `v2.nix` below. It's customary to either have pname + version or only name.

Reply:
Fair enough. :)